ChangeLog.md

```diff
@@ -1,5 +1,10 @@
 # What's new in the WSL Management PowerShell module
 
+## Version 2.1.0 (2023-09-26)
+
+- Added support for retrieving WSL distribution listing from online sources via
+  [`Get-WslDistributionOnline`][] cmdlet.
```

Comment:
There is no matching link for this label, please add one.

Reply:
Updated.  Sorry never done links in Markdown before so had to work it out. :)